lib/run-executable.js

```diff
@@ -21,17 +21,20 @@ function runExecutable(args, cwd, logLevel) {
 
     const childProcess = spawn(pathToExecutable, args, {
       cwd,
-      stdio: ['inherit', 'inherit', 'pipe']
+      stdio: pipe
```

Comment:
Is there a reason for adding this parameter? I'm not sure this level of granularity over the output is necessary and could be confusing as an API.